ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitterSpec.scala

```diff
@@ -289,6 +293,76 @@ class TransactionCommitterSpec
     }
   }
 
+  "setting out of time bounds entry" should {
+
+    "be set" when {
+
+      "a submitting party is not known" in {
+        val context = createCommitContext(
+          recordTime = None,
+          inputs = createInputs(
+            Alice -> Some(hostedParty(Alice)),
+            Bob -> None,
+          ) + (Conversions.configurationStateKey -> None),
```

Comment:
Why do we need to add the configuration key here? Wouldn't it be 'not found' by default?

Reply:
No, if the key is not defined then it means it wasn't requested/created (not part of the input or output) and it would crash